### PR TITLE
Tutorial Update

### PR DESCRIPTION
**Pull request by Joysie**
Content and formatting were improved

### CODE REVIEW

File: tutorials/btp-cf-buildpacks-node-create/btp-cf-buildpacks-node-create.md [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
-description: Create a simple Node.js application in the Cloud Foundry Command Line Interface (cf CLI) and enable services for it.
+title: Create a Node.js Application with Cloud Foundry Node.js Buildpack 
+description: Create a simple application and enable services for it, by using  the Cloud Foundry Node.js Buildpack and Cloud Foundry Command Line Interface (cf CLI).
 auto_validation: true
 time: 40
 tags: [ tutorial>beginner, software-product>sap-btp--cloud-foundry-environment, software-product-function>sap-btp-cockpit]
 primary_tag: programming-tool>node-js
 ---
 
+# Create a Node.js Application with Cloud Foundry Node.js Buildpack
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-buildpacks-node-create/btp-cf-buildpacks-node-create.md:13]: There is already a title in the meta data. Please remove this line.

File: tutorials/btp-cf-buildpacks-node-create/btp-cf-buildpacks-node-create.md [hunk 2]
```diff
@@ -1,33 +1,37 @@
...
+<!-- description --> Create a simple application and enable services for it, by using  the Cloud Foundry Node.js Buildpack and Cloud Foundry Command Line Interface (cf CLI).
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-buildpacks-node-create/btp-cf-buildpacks-node-create.md:14]: There is already a description in the meta data. Please remove this line.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please address my comments and resolve conflicts.
Always do a "pull" for your fork before pushing anything to production.

File: tutorials/btp-cf-buildpacks-node-create/btp-cf-buildpacks-node-create.md (hunk 1 above)
Comment (Joysie) [reply tutorials/btp-cf-buildpacks-node-create/btp-cf-buildpacks-node-create.md]: Done!

File: tutorials/btp-cf-buildpacks-node-create/btp-cf-buildpacks-node-create.md (hunk 2 above)
Comment (Joysie) [reply tutorials/btp-cf-buildpacks-node-create/btp-cf-buildpacks-node-create.md]: Thank you! Done.

**Comment by MichaelCzcz** [comment]
@Joysie Please resolve conflicts.
image

**Pull request merged**